src/platform/shell.go

```diff
@@ -425,21 +425,29 @@ func (env *Shell) Pwd() string {
 
 func (env *Shell) HasFiles(pattern string) bool {
 	defer env.Trace(time.Now(), pattern)
+	pattern = strings.ToLower(pattern)
```

Comment:
you want to move this down below line 430 as you only really need it as of there.

Reply:
Just before the for loop yeh?

Reply:
Just did it